src/test/resources/mockJDK-1.8/jre/lib/annotations.jar_LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
For that reason we need that mock JDK stuff? The are JetBrains annotations on Maven https://mvnrepository.com/artifact/org.jetbrains/annotations/19.0.0

Reply:
Not sure what's with annotations, we used mock JDK for testing (running tests without running the real instance of IDE), because it was the only way to correctly resolve PsiReferences (during testing). But ok, this is for sure redundant for now.

Reply:
The are a more convenient way of testing IntelliJ plugins, see https://www.jetbrains.org/intellij/sdk/docs/basics/testing_plugins/testing_plugins.html

Reply:
I think we are talking about different things. We used ways listed in this link (LightJavaCodeInsightFixtureTestCase, to be exact), we used mockJDK only to initialize JDK in this fixture (because the standard one couldn't resolve methods).